src/main/riddl/Members/members.riddl

```diff
@@ -245,12 +230,6 @@ context MembersContext  is {
               and "set entity lastModifiedOn to today()"
               and "yield event MemberActivated"
             }
-            on command InactivateMember {
-              then error "Cannot set state to Inactive on a member that is not active"
-            }
-            on command SuspendMember {
-              then error "Cannot set state to Suspended on a member that is not active"
-            }
             on command TerminateMember {
```

Comment:
These need to be retained per the comment above about UpdateMemberStatus message